app.rb

```diff
@@ -51,24 +51,30 @@ def create_person
     end
   end
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/203131662-ee5a0c24-2478-4a9e-827b-959bad9f15f6.png)

- [ ] Its great that you have implemented the requested changes but i guess you have broken your application while refactoring your application. As you can see in the image above :arrow_up: while creating a rental your application shows an error :fearful: Kindly fix that :+1: